test/specs/deployAndRetrieve.e2e.ts

```diff
@@ -90,14 +90,8 @@ describe('Deploy and Retrieve', async () => {
   step('Deploy with SFDX: Deploy This Source to Org - ST enabled', async () => {
     const workbench = await (await browser.getWorkbench()).wait();
     // Clear the Output view first.
-    await utilities.openOutputView();
```

Comment:
I'm getting rid of all the occurrences of this as it's actually not needed anymore, because in utilities.getOutputViewText() we're already opening the output view

Reply:
True statement - but I think the comment is placed in the wrong location!  utilities.openOutputView() is not needed here because 'View: Clear Output' does not require first opening the Output panel.